dbt_semantic_interfaces/protocols/saved_query.py

```diff
@@ -0,0 +1,39 @@
+from abc import abstractmethod
+from typing import Optional, Protocol, Sequence
+
+from dbt_semantic_interfaces.protocols.metadata import Metadata
+from dbt_semantic_interfaces.protocols.where_filter import WhereFilter
+
+
+class SavedQuery(Protocol):
```

Comment:
We've got separate work that is to add an optional `label` to many objects (metrics, semantic_models, dimensions, measures). Are we planning to add them to saved queries as well? If so we could do so now. It'll make it easier on the core side of things because we won't have to implement saved queries, and then come back and add a new attribute to them a week later.

Reply:
Not sure about whether it's supposed to be added, but seems fine to.

Reply:
I think we'll just quickly add it in a PR where we add label to everything else. So don't worry about it here.